core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -77,7 +77,9 @@ THE SOFTWARE.
                         <div class="plugin-manager__categories">
                           <j:forEach var="label" items="${p.categories}">
                             <span class="plugin-manager__category-label">
-                              ${app.updateCenter.getCategoryDisplayName(label)}
+                              <a href="?filter=${app.updateCenter.getCategoryDisplayName(label)}">
```

Comment:
Wow, love that these can be made into hyperlinks. I'd say to inspire us in bootstrap pill buttons (https://getbootstrap.com/docs/4.4/components/badge/#links) and maybe style the background and remove the text underline. I can get some styles going on if you want.

Reply:
I already expected something like that, but it's a waste of time for me to start on before any feedback 😆 Go for it 👍

Reply:
Will do, may take a few days if that's OK

Reply:
Can we remove the on-hold label on this one, now that my changes are merged in?

Reply:
Right, thanks for the reminder @fqueiruga